### DISCUSSION TITLE
DC-DC Buck Converter with Arduino Uno - Seyed Amir Alavi | Embedded Software Engineer

### DISCUSSION THREAD

**Discussion by giscus** [samiralavi/samiralavi.github.io#1] [Announcements]
# DC-DC Buck Converter with Arduino Uno - Seyed Amir Alavi | Embedded Software Engineer

This tutorial teaches the basics of DC-DC buck converters with a experimental Arduino-based circuit, for learning purposes.

https://samiralavi.github.io/blog/buck_coverter/buck_converter_arduino/

**Comment by StickFix** [comment]
Hello Sir! I am very grateful to you for having published this article, which shows your care in explaining everything.
Just two questions: the 1N5819 looks to be a transistor (a thing with 3 poles) but I only can find any normal looking diode (2 poles), and can't find some transistor like a 2N5819. Can you please check the part number of this part?
Second question is that I didn't understand where the feedback of the output voltage is, since the existance of the interdependence of the PWM with the ouput load and voltage. The only input to the arduino is the potentiometer to set the output voltage as far as I can see.
Thank you very much again!!
